src/angular/controllers/ctrl-widget-catalog-details.test.js

```diff
@@ -0,0 +1,247 @@
+describe('WidgetCatalogDetailsController', () => {
```

Comment:
Since the widget catalog and details pages have been completely rewritten in React, this file is no longer necessary. I'll make a separate commit to remove it.